scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -27,7 +28,7 @@
 from .process import set_column_map
 
 # These directories are excluded from testing
-_EXCLUDE_DIRS = ['common', 's2201', '__pycache__']
+_EXCLUDE_DIRS = ['common', 's2201', 's0902', '__pycache__']
```

Comment:
pls open an issue to fix tests for 0902 if there isn't one already.

Reply:
Done. Have opened #585.